server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -902,10 +902,11 @@ if(currentReleaseNumber !== futureReleaseNumber){
       ltpConfigurationList.push(tcpClientUuidOfFutureApplication[0]);
     }
 
-    if (httpClientUuidOfCurrentApplication) {
+    if (httpClientUuidOfCurrentApplication !== httpClientUuidOfFutureApplication) {
```

Comment:
kindly also include a check to make sure httpClientUuidOfCurrentApplication is present ..